docs/pages/access-controls/sso/okta.mdx

```diff
@@ -56,7 +56,7 @@ GROUP ATTRIBUTE STATEMENTS
 - Name: `groups` | Name format: `Unspecified`
 - Filter: `Matches regex` |  `.*`
 
-![Configure APP](../../../img/sso/okta/setup-redirection.png)
+<img width="542" alt="setup-redirection" src="https://user-images.githubusercontent.com/117299936/208718729-d16db2ed-b6bb-45b9-a9d7-7f3eecc5bde3.png">
```

Comment:
We should probably be committing the image like we do our other screenshots rather than using GitHub as a CDN.

Reply:
Yes, please overwrite the image in the repo and commit it. I will then optimize the image for web as needed.

Reply:
Would be even better if you optimize the image before committing it, since once the original image is in the repo it never goes away..

Reply:
Yes, that would be ideal. I use [Trimage](https://trimage.org/) but there are plenty of similar tools.

Reply:
> since once the original image is in the repo it never goes away..

OP could also squash the optimized image commit to rewrite history.